doc/site/docs/02-core-concepts/06-body.md

```diff
@@ -142,41 +137,39 @@ class BodyType {
 
 ### MIME type detection
 
-Relic includes intelligent MIME type detection that analyzes content to automatically determine the appropriate content type. This feature works for both text-based and binary content formats, saving you from having to manually specify types in most common scenarios.
+Relic analyzes content to determine the appropriate content type for both text and binary formats. This detection reduces boilerplate in handlers and helps ensure clients interpret responses correctly.
 
 #### Text content detection
 
-For text-based content, Relic examines the structure and syntax to identify common formats. JSON content is detected by looking for characteristic opening braces or brackets:
+JSON is detected by typical structural markers like braces and brackets, so you can return JSON strings and let Relic set the proper MIME type. This keeps your code focused on business logic instead of metadata:
```

Comment:
Show we really discuss the internals of how the detection work? This is not really needed for using the API. Perhaps just mention the formats that we can automatically detect is enough.

Reply:
Noted